integrationtest/backend-tck/src/main/java/org/hibernate/search/v6poc/integrationtest/backend/tck/search/SearchSortIT.java

```diff
@@ -255,6 +256,19 @@ public void separateSort() {
 				.hasReferencesHitsExactOrder( indexName, THIRD_ID, SECOND_ID, FIRST_ID, EMPTY_ID );
 	}
 
+	@Test
+	public void byDistance() {
+		SearchQuery<DocumentReference> query = simpleQuery( b -> b.byDistance( "geoPoint", new ImmutableGeoPoint( 45.757864, 4.834496 ) ) );
+
+		DocumentReferencesSearchResultAssert.assertThat( query )
+				.hasReferencesHitsExactOrder( indexName, FIRST_ID, THIRD_ID, SECOND_ID, EMPTY_ID );
+
+		query = simpleQuery( b -> b.byDistance( "geoPoint", 45.757864, 4.834496 ) );
+
+		DocumentReferencesSearchResultAssert.assertThat( query )
+				.hasReferencesHitsExactOrder( indexName, FIRST_ID, THIRD_ID, SECOND_ID, EMPTY_ID );
```

Comment:
Maybe comment here, explaining why we don't test descending sorts?

Reply:
Done.